src/managers/GuildBanManager.js

```diff
@@ -64,7 +85,7 @@ class GuildBanManager extends BaseManager {
    *   .catch(console.error);
    */
   fetch(options) {
-    if (!options) return this._fetchMany();
+    if (!options) return this._fetchMany(options.cache);
```

Comment:
```suggestion
    if (!options) return this._fetchMany();
```
If options is a falsy value, how would it have the cache property on it?

Reply:
fixed in a129943